src/main/java/hunternif/mc/atlas/item/ItemEmptyAtlas.java

```diff
@@ -14,7 +16,10 @@
 import net.minecraftforge.common.ForgeHooks;
 
 public class ItemEmptyAtlas extends Item {
+
 	public ItemEmptyAtlas() {
+	    setUnlocalizedName("emptyAntiqueAtlas");

```

Comment:
I'm moving, so I won't be able to look at anything until next week.  The gradle plugin sounds cool.